code/modules/shuttle/marine_dropship.dm

```diff
@@ -67,10 +67,10 @@
 	var/turf/left = locate(C.x - leftright, C.y, C.z)
 	var/turf/right = locate(C.x + leftright, C.y, C.z)
 
-	explosion(front, 1, 5, 9, 0)
-	explosion(rear, 3, 6, 10, 0)
-	explosion(left, 3, 6, 10, 0)
-	explosion(right, 3, 6, 10, 0)
```

Comment:
Do you need to also update the canterbury crash explosions?

Reply:
I would've thought the delayed nature of the explosions is what killed the xenos, rather than the size / spread

Reply:
It was actually the size.
I need to update all explosions because we no longer signalize no-explosion with `-1`, we now do it with `0`, so the first values need to be increased in general.